examples/print.ts

```diff
@@ -17,9 +18,18 @@ switch (x) {
 }
 `;
 
-console.log(
-  print(parse(code, { target: "es2020", syntax: "typescript" }), {
-    minify: true,
-    isModule: true,
-  }).code,
-);
+const ast = parse(code, { target: "es2019", syntax: "typescript", comments: true });
+const regeneratedCode = print(ast, {
+  minify: true,
+  module: {
+    type: "commonjs"
+  },
+}).code;
+
+console.log(code);
+console.log(''); console.log(''); console.log('');
+console.log(ast);
+console.log(''); console.log(''); console.log('');
+console.log(regeneratedCode);
```

Comment:
Please keep the previous examples

Reply:
Removed the unnecessary console logging. I left the changes required to compile. And a small refactor to support a stackoverflow answer (code, ast, regeneratedCode). Note: es2020 isn't a valid enum type and `isModule: true` no longer exists. Let me know if it is or feel free to add changes.

PR to a fork is new to me. Sorry for the numerous commits and forgetting to lint! I will make sure it is draft until I am ready.

Reply:
Updated to be closer to the original again. The link to example print.ts should be sufficient if someone wants a more detailed version.